### PR TITLE
fix results page mobile styling and fonts

### PR DESCRIPTION
**Pull request by MadelinTom**
https://trello.com/c/2toRvPAK/127-mobile-styling-of-the-page-of-results

### CODE REVIEW

**Comment by matejoslav** [comment]
@MadelinTom looks good but we should be using styled components everywhere

**Review by matejoslav — CHANGES_REQUESTED** [review]
see above

File: web-app/src/components/resultsPage/ResultCard.js [hunk 1]
```diff
@@ -6,10 +6,19 @@ export const ResultCard = ({ title, body, id = "result card" }) => {
...
+              margin: "10px",
+              fontFamily: "Playfair Display",
+              fontSize: "16px",
+              fontWeight: "bold",
+            }}
+          >
             <b>{title}</b>
           </h4>
-          <p style={{margin: "10px"}}>{body}</p>
+          <p style={{ margin: "10px", fontFamily: "Rleway", fontSize: "16px" }}>
```
Comment (matejoslav) [inline web-app/src/components/resultsPage/ResultCard.js:19]: typo in fontFamily entry

**Review by matejoslav — CHANGES_REQUESTED** [review]
Found a small typo, also do we still need the font changes? Maybe if we merge dev into the branch we can get rid of them

**Review by matejoslav — APPROVED** [review]

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210228-121757 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20210228-121757") to [dev](https://console.bluemix.net/apps/4a65172b-6ccd-4969-a77b-771a8adf5eb5?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit abbe7043eb1aaf33791d9f87080c2a579107a4db

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210302-130622 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210302-130622") to [test](https://console.bluemix.net/apps/bef762e5-4952-4e16-a258-78b99b9fb66d?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit abbe7043eb1aaf33791d9f87080c2a579107a4db

**Pull request merged**